engine/src/main/scala/cromwell/engine/io/gcs/GcsBatchCommandContext.scala

```diff
@@ -73,14 +73,21 @@ final case class GcsBatchCommandContext[T, U](request: GcsBatchIoCommand[T, U],
   /**
     * Queue the request for batching
     */
-  def queue(batchRequest: BatchRequest) = request.operation.queue(batchRequest, callback)
+  def queue(batchRequest: BatchRequest) = {
+    request.customDebug(s"GcsBatchCommandContext.queue '$batchRequest'")
+    request.operation.queue(batchRequest, callback)
+  }
 
   /**
     * On success callback. Transform the request response to a stream-ready response that can complete the promise
     */
-  private def onSuccessCallback(response: U, httpHeaders: HttpHeaders) = handleSuccessOrNextRequest(request.onSuccess(response, httpHeaders))
+  private def onSuccessCallback(response: U, httpHeaders: HttpHeaders) = {
+    request.customDebug(s"GcsBatchCommandContext.onSuccessCallback '$response'")
```

Comment:
TOL good call not logging the headers 😉

Reply:
Do we potentially want a `toPrettyElidedString` on the response?

Reply:
I want the whole response. For now. I may regret that later.